code/game/g_active.cpp

```diff
@@ -2904,8 +2904,8 @@ qboolean G_CheckClampUcmd( gentity_t *ent, usercmd_t *ucmd )
 							damage = Q_irand( 3, 5 );
 						}
 					}
-					if ( ent->client->ps.legsAnimTimer <= 1150
-						&& ent->client->ps.legsAnimTimer > 10500 )
+					if ( ent->client->ps.legsAnimTimer > 1150
```

Comment:
It seems this conflicts with #1119. I'm not sure which one is correct

Reply:
If you look at the other checks in the same function you will see that most are checking for a 100ms time window such as:
```
if ( ent->client->ps.legsAnimTimer <= 3150
    && ent->client->ps.legsAnimTimer > 3050 )
```
or
```
if ( ent->client->ps.legsAnimTimer <= 5700
    && ent->client->ps.legsAnimTimer > 5600 )
```
which is why I assumed mine would be the logical fix here. I could also be wrong though.